src/main/java/frc/robot/commandGroups/EjectCoralThenZero.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.commandGroups;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.ElevatorCommands.ZeroElevator;
+import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.FunnelSubsystem;
+import frc.robot.subsystems.TootsieSlideSubsystem;
+
+public class EjectCoralThenZero extends SequentialCommandGroup {
+  public EjectCoralThenZero(ElevatorSubsystem elevatorSubsystem, TootsieSlideSubsystem tootsieSlideSubsystem, FunnelSubsystem funnelSubsystem) {
+    Command a = null;
+    if (funnelSubsystem.drakeTripped()) { 
```

Comment:
Rather than check the sensor we should just run both commands regardless, just in case. Simplifies this as well. Should allow it to be easily declared inline:


`new SequentialCommandGroup(EjectCoralFR(...), ZeroElevator(...)); `

Rather than needing its own file.